obstacle_range: 5.0
raytrace_range: 6.0

robot_radius: 0.6
transform_tolerance: 0.3

# Definimos ambas fuentes de datos
observation_sources: raw_scan semantic_scan

raw_scan:
  data_type: LaserScan
  topic: /raw_scan
  marking: true
  clearing: true           # se limpia r√°pido
  min_obstacle_height: 0.0
  max_obstacle_height: 4.0
  obstacle_range: 5.0
  raytrace_range: 6.0

semantic_scan:
  data_type: LaserScan
  topic: /semantic_scan
  marking: true
  clearing: true          # persisten las bandas densas
  min_obstacle_height: 0.0
  max_obstacle_height: 4.0
  obstacle_range: 5.0
  raytrace_range: 5.0      # no limpia por raytrace
